sdk/storage/azure-storage-blob/azure/storage/blob/_shared/response_handlers.py

```diff
@@ -144,7 +144,7 @@ def process_storage_error(storage_error):
     error = raise_error(message=error_message, response=storage_error.response)
     error.error_code = error_code
     error.additional_info = additional_data
-    raise error
+    raise error.raise_with_traceback()
```

Comment:
I thought `raise_with_traceback` already called `raise` - does it not?

Reply:
you are absolutely correct